theseus/geometry/so3.py

```diff
@@ -84,10 +84,15 @@ def exp_map(tangent_vector: torch.Tensor) -> LieGroup:
         theta = torch.linalg.norm(tangent_vector, dim=1, keepdim=True).unsqueeze(1)
         theta2 = theta ** 2
         # Compute the approximations when theta ~ 0
-        sel = theta >= 0.005
-        a = torch.where(sel, theta.cos(), 8 / (4 + theta2) - 1)
-        b = torch.where(sel, theta.sin() / theta, 0.5 * a + 0.5)
-        c = torch.where(sel, (1 - a) / theta2, 0.5 * b)
+        small_theta = theta < 0.005
```

Comment:
What's the motivation for this flip in the indexing? (asking only out of curiosity, this looks fine to me).

Reply:
Just for the consistency with SE2.